src/app/[rc_id]/sources/page.tsx

```diff
@@ -17,21 +17,29 @@ export default async function Home(
   );
 }
 
-async function ListSourceData({ rc_id }: { rc_id: string }) {
+async function ListSourceData({ rc_id }: { rc_id: number }) {
   const api: TServerComponentAPI = serverContainer.get(TRPC.REACT_SERVER_COMPONENTS_API);
```

Comment:
Sorry if this is a silly or premature question, but why are we going through a controller on the conversations page, but sticking with a direct TRPC call on this page? If there's a good reason, and it's just not obvious to me, is it worth putting in comments to help remind us of this logic if in the future we come back to this code?

Reply:
@ydamit don't take this one as an example, code in the app side for the pages is really dirty and needs quite a bit of refactoring :p 
What should happen here based on our latest conventions is that we instantiate the `listSourceDataController` (the client one) with `clientContainer.get(...)`